### PR TITLE
fix #51140 ww1.solarmovie.fun

### LINKED ISSUE #51140: ww1.solarmovie.fun
### Issue URL (Annoyance)
 [https://ww1.solarmovie.fun/movie/contagion/u0KTHMQB-watch-free-online.html](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fww1.solarmovie.fun%2Fmovie%2Fcontagion%2Fu0KTHMQB-watch-free-online.html)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | Ext
AdGuard version: | 3.3.8
Browser: | Chrome
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> AdGuard Spanish/Portuguese

### PR DESCRIPTION
**Pull request by Sergey-Lyapin**
#51140

### CODE REVIEW

File: AnnoyancesFilter/sections/annoyances.txt [hunk 1]
```diff
@@ -1331,6 +1333,8 @@ slate.com,businessinsider.de,fastcompany.com,heise.de,history.com,itprotoday.com
 bournemouthecho.co.uk,eveningtimes.co.uk,greenocktelegraph.co.uk,heraldscotland.com,swindonadvertiser.co.uk,wiltshiretimes.co.uk##.cta-component-full-width
 !
 statut.by##a[title="Telegram"]
+solarmovie.fun#?#footer > span:contains(Subscribe)
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/annoyances.txt]: better leave footer as is

File: AnnoyancesFilter/sections/antiadblock.txt [hunk 2]
```diff
@@ -288,6 +288,7 @@ augsburger-allgemeine.de,4fansites.de,general-anzeiger-bonn.de,kurier.de,knuddel
 rezka.ag,hdrezka.host,hdrezka-online.me,hdrezka.com.ua,hdrezka.site,hdrezkaag.net,hdrezka.website,hdrezka.tv,hdrezka.today,hdrezka.lol,hdrezka.name,hdrezka.co,hdrezka.club,hdrezka.cm,tryhdrezka.com,hdrezkayou.com,hdrezka-ag.com#%#//scriptlet("set-constant", "window.abc", "false")
 !
 upflix.pl#$#.ad.banner { display: block!important; position: absolute!important; left: -3000px!important; }
+solarmovie.fun#?#main > div[class] > div[class] > section[class] > div.hhotnews:contains(AdBlocker)
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/antiadblock.txt]: works without ext-CSS
`solarmovie.fun##.hhotnews`

File: AnnoyancesFilter/sections/popups.txt [hunk 3]
```diff
@@ -92,6 +92,7 @@ mycredit.ua#$#.js-modal-show { display: none !important; }
 mycredit.ua#$#.modal-backdrop { display: none !important; }
 mycredit.ua#$#body { overflow: visible !important; }
 mycredit.ua#$#.wrapper { -webkit-filter: none !important; filter: none !important; }
+solarmovie.fun##.banner-wrapper
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/popups.txt]: I don't see

File: EnglishFilter/sections/specific.txt [hunk 4]
```diff
@@ -191,6 +191,7 @@ missoulian.com,kenoshanews.com,cumberlink.com,trib.com,dailyprogress.com,stltoda
 /^(?!.*(google\.com|gstatic\.com|googleapis\.com|shrink\.pe|recaptcha.net)).*$/$third-party,script,domain=aii.sh|uptoshort.com|gsurl.in
 !
 ashemaletube.com##.vast-overlay
+solarmovie.fun##a[href^="https://weedzy.co.uk/"] > img
```
Comment (Alex-302) [inline EnglishFilter/sections/specific.txt]: I don't see

Comment (Sergey-Lyapin) [reply EnglishFilter/sections/specific.txt]: looks like they don't advertise weed anymore

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**